Update ez-today.typ

iamanro opened the pull request:
Added Slovakian (sk) and Polish (pl) version.

iamanro committed:
Update ez-today.typ

Added Slovakian (sk) and Polish (pl) version.

CarloSchafflik12 reviewed (commented):
Hello, thank you.
I,m not an expert on the Slovak language, but is October really "Okt"?

iamanro commented:
Yes. Source for example 1st row "Slovensky":
https://mesiace.sk/

iamanro committed:
Update ez-today.typ

Corrected sk and pl from nominativ to genitiv.
+ support for format:"ISO" as yyyy-mm-dd
